libraries/ms-common/src/stm32f0xx/wait.c

```diff
@@ -0,0 +1,7 @@
+#include "wait.h"
+
+#include "stm32f0xx.h"
+
+void wait(void) {
+  __WFE();
```

Comment:
[I think we might want to use WFI?](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.faqs/ka15473.html) [Another page on WFI vs WFE](https://community.arm.com/processors/f/discussions/5695/difference-between-wfi-and-wfe)